src/web/components/global-nav/container/sub-nav/sub-nav.js

```diff
@@ -19,10 +21,18 @@ class SubNav extends Core {
 
     setModuleIndicatorName(name){
         this._findDOMEl('.hig__global-nav__container__sub-nav__left', this.el).textContent = name;
+        this._findDOMEl('.hig__global-nav__container__sub-nav__left.hig__global-nav__container__sub-nav__spacer', this.el).textContent = name;
```

Comment:
tldr; I'm duplicating the module name on the right side in order to
- have the tabs be centered on a wide screen
- Have the tabs stack next to the module name on a narrow screen

If you can think of a less hacky approach I'm all ears 😂